test/swarm.js

```diff
@@ -1,82 +1,82 @@
-var hat = require('hat')
-var Swarm = require('../lib/swarm')
-var test = require('tape')
+// var hat = require('hat')
```

Comment:
Might want to delete this file. Or are you planning to port the tests to use `torrent` instead of `swarm` later?